### PR TITLE
fixed image path and added markdown syntax in markdown-style-guide.md file

### PR DESCRIPTION
**Pull request by xkeshav**
## Changes

- There are wrong image path in content/blog/markdown-style-guide.md file which comes as defult blog page 
- also added syntax and output on the same file, otherwise it was not showing the syntax but render it as markdown, so user unable to understand the exact syntax.

### CODE REVIEW

**Comment by xkeshav** [comment]
When  this PR would be merged? any criteria missing then let me know.

**Comment by ematipico** [comment]
@withastro/maintainers-docs, can anyone have a look at this, please?

File: examples/blog/src/content/blog/markdown-style-guide.md [hunk 1]
```diff
@@ -31,18 +31,43 @@ Itatur? Quiatae cullecum rem ent aut odis in re eossequodi nonsequ idebis ne sap
...
 
-![This is a placeholder image description](/blog-placeholder-1.jpg)
+#### Syntax
+
+```markdown
+![Alt text](./full/or/relative/path/of/image)
+```
+ 
+#### Output
+  ![blog placeholder](../../../public/blog-placeholder-about.jpg)
```
Comment (ElianCodes) [inline examples/blog/src/content/blog/markdown-style-guide.md:41]:
```suggestion
#### Output

![blog placeholder](../../../public/blog-placeholder-about.jpg)
```

File: examples/blog/src/content/blog/markdown-style-guide.md [hunk 2]
```diff
@@ -51,12 +76,43 @@ The blockquote element represents content that is quoted from another source, op
 
 ## Tables
 
+#### Syntax
+
+```markdown
+| Italics   | Bold     | Code   |
+| --------- | -------- | ------ |
+| _italics_ | **bold** | `code` |
+
```
Comment (ElianCodes) [inline examples/blog/src/content/blog/markdown-style-guide.md:85]: empty line removal

```suggestion
| Italics   | Bold     | Code   |
| --------- | -------- | ------ |
| _italics_ | **bold** | `code` |
```

**Review by ElianCodes — COMMENTED** [review]
Just some small nits; but approved!

**Comment by ElianCodes** [comment]
Looks good to me! Let's get this merged 🚀⛴️

**Pull request merged**